cypress/integration/collections_spec.js

```diff
@@ -51,9 +51,10 @@ describe('Dashboard Collections Page', () => {
       cy.visit('/collections');
       cy.wait('@getCollections');
       let i = 0;
-      while (i < cmrFixtureIdx) cy.wait(`@cmr${i++}`);
+
       cy.get('table tbody tr').its('length').should('be.eq', 5);
 
+      while (i < cmrFixtureIdx) cy.wait(`@cmr${i++}`, {timeout: 25000});
```

Comment:
this is kind of a last ditch effort, I'm not sure why sometimes these fixtures aren't getting fired.